src/pages/account/sign-in.tsx

```diff
@@ -17,7 +17,7 @@ const SignInPage: FunctionComponent<SignInPageProps> = ({
 };
 
 export const getServerSideProps: GetServerSideProps = async () => {
-  const callbackUrl = process.env.NEXTAUTH_URL + PageURLs.updateAccount;
+  const callbackUrl = process.env.NEXTAUTH_URL + PageURLs.accountHome;
   return { props: { callbackUrl } };
 };
 
```

Comment:
when we log in we are always directed to the account homepage, it is the account homepage which is responsible for determining if your account is in suitable condition.